posts/2021-03-24-microprofile40-open-liberty-21003.adoc

```diff
@@ -0,0 +1,442 @@
+
+[source, java]
+----
+@APIResponseSchema(MyResponseObject.class)
+----
+
+[#mpopenapischeme]
+==== The "mp.openapi.schema." property prefix
+The `mp.openapi.schema.*` MicroProfile Config property prefix has been added to allow you to define the schema for Java classes using configuration rather than code. The use of this property is functionally equivalent to the use of the `@Schema` annotation on a Java class, but may be used in cases where you do not have access to the source code of a class. The remainder of the property key must be the fully-qualified class name and the value must be a valid OpenAPI schema object, specified in the JSON format.
+For example, in the case where an application needs to represent dates in epoch milliseconds, the following configuration could be used (line escapes and indentation added for readability):
+
+[source, java]
+----
+mp.openapi.schema.java.util.Date = { \
+   "name": "EpochMillis" \
+   "type": "number", \
+   "format": "int64", \
+   "description": "Milliseconds since January 1, 1970, 00:00:00 GMT" \
+ }
+----
+
+Enable OpenAPI 2.0 in the `server.xml`, along with any other features you’re using.
+
+[source, xml]
+----
+<featureManager>
+    <feature>mpOpenAPI-2.0</feature>
+</featureManager>
+----
+
+For more information:
+
+* link:{url-prefix}/guides/microprofile-openapi.html[Guide: Documenting RESTful APIs]
+* link:https://download.eclipse.org/microprofile/microprofile-open-api-2.0/apidocs/[Javadoc]
+* link:https://download.eclipse.org/microprofile/microprofile-open-api-2.0/microprofile-openapi-spec-2.0.html#release_notes_20[Release notes]
+
+
+[#rest-client-2.0]
+== Awesome new features in MicroProfile Rest Client 2.0
+
+MicroProfile REST Client is a type-safe client API enabling rapid development of applications capable of consuming RESTful services. Version 2.0 is the latest update and adds support for HTTP proxy servers, automatically following HTTP redirects, Server Sent Events, and additional configuration options for JSON-B providers and multiple query parameters.
+
+Shortly we will be publishing a dedicated blog post solely on MicroProfile Rest Client 2.0, which will demonstrate all the new features and how to use them.
+
+[#jwt-1.2]
+=== Enhancements in MicroProfile JWT Auth 1.2
+
+MicroProfile JWT 1.2 simplifies the configuration for managing the validation of the JWT by introducing new MicroProfile Config properties. Enhanced signature algorithm support is added in this Open Liberty implementation.
+
+MicroProfile JWT 1.2 adds a number of MicroProfile Config properties to control different aspects of the JWT validation. The specification also adds support for the ES256 signature algorithm, while this Open Liberty beta supports using the RS384, RS512, HS384, HS512, ES256, ES384, and the ES512 signature algorithms.
+
+MicroProfile JWT 1.2 also supports the JWT specified in a cookie.
+
+Watch this space for a more detailed blog on MicroProfile JWT 1.2.
+
+== Summary
```

Comment:
I think we should add a shortcut here and link it to the list aswell since it's a h2 heading

Reply:
```suggestion
[#summary]
== Summary
```